config/locales/en.yml

```
@@ -121,8 +121,8 @@ en:
             base:
               duplicate: "This course already exists. You should add further locations for this course to the existing profile in Publish"
               visa_sponsorship_not_publishable: "You must say whether you can sponsor visas"
-              provider_ukprn_not_publishable: "You must provide a UK provider reference number (UKPRN)"
-              provider_ukprn_and_urn_not_publishable: "You must provide a UK provider reference number (UKPRN) and URN"
+              provider_ukprn_not_publishable: "You must provide a UK Provider Reference Number (UKPRN)"
+              provider_ukprn_and_urn_not_publishable: "You must provide a UK Provider Reference Number (UKPRN) and URN"
```

Trying to move away from using ‘You must’ in validation strings:

```suggestion
              provider_ukprn_not_publishable: "Enter a UK Provider Reference Number (UKPRN)"
              provider_ukprn_and_urn_not_publishable: "Enter a UK Provider Reference Number (UKPRN) and URN"
```

---

Thanks @paulrobertlloyd. I've updated in 9b78dbb